### ISSUE TITLE
Explore CrossEncoders

### ISSUE THREAD

**Issue by thefirebanks** [wri-dssg/policy-data-analyzer#74]
As specified in this issue: https://github.com/UKPLab/sentence-transformers/issues/350#issuecomment-757687915

The authors of SBERT recommend using CrossEncoders for sentence classification.

**Comment by thefirebanks** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Explore CrossEncoders](https://trello.com/c/Je7zGlBt/74-explore-crossencoders)

**Comment by thefirebanks** [comment]
Relevant links:

- CrossEncoder folders: https://github.com/UKPLab/sentence-transformers/tree/master/sentence_transformers/cross_encoder
- CrossEncoder usage: https://github.com/UKPLab/sentence-transformers/blob/master/examples/applications/cross-encoder/cross-encoder_usage.py
- Main threads explaining how/why use CrossEncoders for our example:
    - https://www.sbert.net/examples/applications/cross-encoder/README.html
    - https://github.com/UKPLab/sentence-transformers/issues/350
- Pretrained CrossEncoders: https://www.sbert.net/docs/pretrained_cross-encoders.html
- CrossEncoders and fine-tuning?: https://github.com/UKPLab/sentence-transformers/issues/615
- Retrieve and rerank pipeline: https://www.sbert.net/examples/applications/retrieve_rerank/README.html 
- Advantages of SBERT over BERT: 
    - https://github.com/UKPLab/sentence-transformers/issues/354
    - https://github.com/UKPLab/sentence-transformers/issues/540
- Augmented SBERT:
   - Paper: https://arxiv.org/pdf/2010.08240.pdf
   - Clarifications on steps: https://github.com/UKPLab/sentence-transformers/issues/553
- _Optional_: Batch hard triplet loss: https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/other/training_batch_hard_trec.py
